common/src/test/scala/uk/ac/wellcome/platform/archive/common/fixtures/StorageManifestVHSFixture.scala

```diff
@@ -31,11 +31,7 @@ trait StorageManifestVHSFixture extends EitherValues {
   def storeSingleManifest(
     vhs: StorageManifestDao,
     storageManifest: StorageManifest): Either[StorageError, Unit] =
-    vhs.update(
-      ifNotExisting = storageManifest
-    )(
-      ifExisting = _ => throw new RuntimeException("VHS should be empty!")
-    )
+    vhs.put(storageManifest)
```

Comment:
Honestly now I’d just ditch this helper and in-line this everywhere, it’s shorter!

Or assert that it succeeds as part of this helper.

Reply:
Same as above - i'd like to keep these interfaces in place for now.